src/web3py/extensions/lido_validators.py

```diff
@@ -107,6 +111,22 @@ def get_lido_validators(self, blockstamp: BlockStamp) -> list[LidoValidator]:
         lido_keys = self.w3.kac.get_all_lido_keys(blockstamp)
         validators = self.w3.cc.get_validators(blockstamp)
 
+        no_operators = self.get_lido_node_operators(blockstamp)
+
+        # Make sure that used keys fetched from Keys API >= total amount of total deposited validators from Staking Router
+        total_used_keys = 0
+        for key in lido_keys:
+            if key.used:
+                total_used_keys += 1
```

Comment:
Why don't we query only used keys from KAPI instead? We don't use the other ones anywhere

Reply:
Yeah, good point

Reply:
Does it mean that we will never get `user: false` key from Keys API or it means that we have to filter keys on provider side (`src/providers/keys/client.py`)?

Reply:
Yeah, pass ?used=false to keys service API

Reply:
> Yeah, pass ?used=false to keys service API

`?used=true` I suppose.
OK, got it.

Reply:
Done